wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotController.java

```diff
@@ -241,4 +241,13 @@ public static CANStatus getCANStatus() {
     CANJNI.getCANStatus(status);
     return status;
   }
+
+  /**
+   * Get the serial number of the robot controller.
+   *
+   * @return The serial number of the robot controller
+   */
+  public static String getSerialNumber() {
+    return System.getenv("SERIALNUM");
```

Comment:
It's lowercase "serialnum".

Reply:
This and getComments() should be implemented in the HAL with appropriate hooks in the simulated HAL so it's possible to unit test them in simulation.